core/src/wallet/pool/WalletPool.cpp

```diff
@@ -106,6 +105,9 @@ namespace ledger {
             _publisher = std::make_shared<EventPublisher>(getContext());
         }
 
+        WalletPool::~WalletPool() {
```

Comment:
Isn't this the same behavior as default constructor?

Reply:
It is, but… that’s the previous code (I just moved it around). I code remove that and put a `= default` if you prefer (I do prefer, too).